.github/workflows/test.yml

```diff
@@ -0,0 +1,27 @@
+name: Tests
+on: [push, pull_request]
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        entry:
+          - { ruby: "2.7" }
+          - { ruby: "3.0" }
+          - { ruby: "3.1" }
+          - { ruby: ruby-head, ignore: true }
+          - { ruby: jruby-head, ignore: true }
+    name: test (ruby=${{ matrix.entry.ruby }}, concurrency=${{ matrix.entry.concurrency || 'none' }})
```

Comment:
you still have a reference to concurrency in the name here, remove

Reply:
🤦 lol sorry about that. I gotta read a little more carefully while having ☕